Packs/ApiModules/Scripts/HTTPFeedApiModule/HTTPFeedApiModule_test.py

```diff
@@ -221,6 +221,7 @@ def test_feed_main_fetch_indicators(mocker, requests_mock):
             'asndrop_country': 'US',
             'asndrop_org': 'LAKSH CYBERSECURITY AND DEFENSE LLC',
             'tags': tags.split(','),
+            'trafficlightprotocol': None,
```

Comment:
I see that you've updated the unit test because now you are populating the tlp field even when its value is `None` (i.e. the user didn't set a value in the integration instance configuration) - can you check if this impacts what you see when viewing an indicator in the UI? If it is shown there, then I think this should be changed so that we only populate the field when it isn't `None`.